src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -33,22 +33,128 @@
+                                                    </button>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                }
+                                else
+                                {
+                                    @if (Model.PreferredApprovers.Contains(User.GetGitHubLogin()))
+                                    {
+                                        <button type="submit" class="btn btn-sm border shadow-sm btn-success">
+                                            <i class="fa-solid fa-circle-check"></i>&nbsp;&nbsp;Approve for GA
+                                        </button>
+                                        itemRendered = true;
+                                    }
+                                    else
+                                    {
+                                        <button type="submit" class="btn btn-sm border shadow-sm btn-success" disabled>
+                                            <i class="fa-solid fa-circle-check"></i>&nbsp;&nbsp;Cannot Approve for GA
+                                        </button>
+                                    }
+                                }
+                            }
+                        }
+                        else
+                        {
+                            @if (Model.Revision.Approvers.Contains(User.GetGitHubLogin()))
+                            {
+                                <button type="submit" class="btn btn-sm shadow-sm btn-light" disabled data-placement="bottom" data-toggle="tooltip" title="API review cannot be approved when comparing against unapproved revision.">
+                                    <i class="fa fa-sm fa-undo" aria-hidden="true"></i>&nbsp;&nbsp;Revert API Approval
+                                </button>
+                            }
+                            else
+                            {
+                                <button type="submit" class="btn btn-sm shadow-sm btn-light" disabled data-placement="bottom" data-toggle="tooltip" title="API review cannot be approved when comparing against unapproved revision.">
+                                    <i class="far fa-thumbs-up" aria-hidden="true"></i>&nbsp;&nbsp;Approve API
+                                </button>
+                            }
+                            itemRendered = true;
+                        }
+                    </form>
+                    @if (itemRendered)
+                    {
+                        <small class="form-text text-muted mx-4 my-1">Approval for Current Revision of the API.</small>
+                        <div class="dropdown-divider"></div>
+                        itemRendered = false;
                    }

```

Comment:
```suggestion
                        itemRendered = false;
                    }
```

Reply:
Its strange but I don't see this on my local machine. I wonder what's going on here.